src/StreamrClient.js

```diff
@@ -171,7 +173,11 @@ export default class StreamrClient extends EventEmitter {
                 })
 
             // Check pending publish requests
-            this.publisher.sendPendingPublishRequests()
+            const publishQueueCopy = this.publishQueue.slice(0)
+            this.publishQueue = []
+            publishQueueCopy.forEach((args) => {
+                this.publish(...args).catch((err) => { throw err })
```

Comment:
`.catch((err) => { throw err })` doesn't actually do anything? Also uncaught promise reject here I guess? Should this be something like:
```
.catch((err) => {
    // log error
    // emit error event on client: this.emit('error', 'error message...')
})
```